client/js/options.js

```diff
@@ -22,156 +36,269 @@ const options = {
 	notifyAllMessages: false,
 	showSeconds: false,
 	statusMessages: "condensed",
-	theme: $("#theme").attr("href").replace(/^themes\/(.*).css$/, "$1"), // Extracts default theme name, set on the server configuration
+	theme: $theme.attr("href").replace(/^themes\/(.*).css$/, "$1"), // Extracts default theme name, set on the server configuration
```

Comment:
Can you change all `.attr` to `.prop` please, this was done on master.

Reply:
Yeah sure. 

Out of curiosity, why has it been changed?

Reply:
https://github.com/thelounge/lounge/pull/2015#discussion_r164976035

Reply:
Changed it. Also rebased and squashed it.